WooCommerce/UITestsFoundation/Screens/Products/SingleProductScreen.swift

```diff
@@ -46,4 +46,30 @@ public final class SingleProductScreen: ScreenObject {
         XCTAssertTrue(app.staticTexts["TIP"].exists)
         XCTAssertTrue(app.textViews[productName].exists)
     }
+
+    public func verifyProductTypeScreenLoaded(productType: String) throws -> Self {
+        let priceLabel = NSPredicate(format: "label CONTAINS[c] %@", "price")
```

Comment:
WDYT of renaming `priceLabel` to `addPriceLabel`, as done for `addVariationLabel`?

Reply:
updated in https://github.com/woocommerce/woocommerce-ios/pull/8407/commits/c9a64bce98ca8c9a44005cc30b3e2e8e5a22e02b